### ISSUE TITLE
Improve Quote block a11y

### ISSUE THREAD

**Issue by mbmjertan** [infinum/eightshift-frontend-libs#690]
### Describe your bug

Quote block is sometimes read as a figure, and sometimes as text by VoiceOver. This should be as consistent as possible, and always cited properly.

> VoiceOver doesn't read the full text in the Quote block, only the first word. It seems it uses the content of the second line in the quote block as the figure description.
> 
> Additionally, in Firefox and in Safari, it treats the block as a figure, reading the text of the second line as description of the figure. In Chrome, it reads the elements in the block one by one.

### Steps to Reproduce

Read the quote block in Firefox, Safari and Chrome with VoiceOver.

### Expected behavior

To define with @lljevako.

### Please confirm that you have searched existing issues in this repo.

Yes

### Please confirm that you have searched in our documentation and couldn't find the answer.

Yes

### Please confirm that your bug occurs with all plugins uninstalled and with the default WordPress theme active.

Yes
